### DISCUSSION TITLE
Add social logins

### DISCUSSION THREAD

**Discussion by code-guru1** [bageldb/libraries#4] [Q&A]
Hey, I'm using BagelDB with Vuejs. Currently, I am using the login/password authentication (https://docs.bageldb.com/bagelAuth-api/#user-login) and it is working fine and I am planning to add social logins via Firebase in my Vuejs project. but I am not sure how to do it with the project since it uses authentication from BagelDB. Is there a way to use both? 
Suggest me how we can do it in a better way.

**Comment by ghost** [comment]
Hey @adilmahmoodcs,

I haven't tried to link other auth providers with BagelDB's authentication, but while looking into this possibility I found [this](https://firebase.google.com/docs/auth/web/account-linking#link-email-address-and-password-credentials-to-a-user-account) in firebase's docs.

It seems that you might need to create a BagelAuth and firebase user using the same email and password simultaneously in order to link both. 

Follow up with us and let us know how it works out.
